devfiles/cpp/devfile.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: cpp
+projects:
+  -
+    name: cpp-hello-world
+    source:
+      type: git
+      location: 'https://github.com/che-samples/cpp-hello-world'
+components:
+-
+  type: chePlugin
+  id: che-incubator/cpptools/latest
+  alias: cpp-plugin
+  preferences:
+    clangd.path: /usr/bin/clangd
+    cdt.clangd.binaries.enable: false
+-
+  type: dockerimage
+  alias: cpp-dev
+  image: registry.centos.org/centos/devtoolset-7-toolchain-centos7
```

Comment:
We should not use a community image here, it has the same issues as all others (no entry in `/etc/passwd`, no writable `$HOME` directory, etc.)